### PR TITLE
Testing Sergey Kintsel's change to upgrade packages to make it work with stack

### PR DESCRIPTION
**Pull request by danstiner**

### CODE REVIEW

File: app/Main.hs [hunk 1]
```diff
@@ -4,19 +4,19 @@ module Main
   ( main
   ) where
 
-import           Actions
```
Comment (sergey-kintsel) [inline app/Main.hs]: tbh I am not sure why this change appeared because I use hfmt as a plugin for VS Code and it formatted the code in stylish-haskell style

Comment (danstiner) [reply app/Main.hs]: Interesting. I do have a config file `.stylish-haskell.yaml` that hfmt uses when I invoke it from the command line at least, maybe the working directory is not set to the project directory when hfmt is invoked from the plugin? That shouldn't be necessary once #22 is fixed, but sorry that's the way it is right now.

File: src/Language/Haskell/Source/Enumerator.hs [hunk 2]
```diff
@@ -29,7 +29,7 @@ enumPath path = do
 enumPackage :: FilePath -> ConduitT () FilePath IO ()
 enumPackage cabalFile = readPackage cabalFile >>= expandPaths
   where
-    readPackage = lift . readGenericPackageDescription Verbosity.silent
```
[the remark this answers is not in the document]
Comment (danstiner) [reply src/Language/Haskell/Source/Enumerator.hs]: Opened an issue to circle back on this, but I don't want to block merging: #37

**Comment by danstiner** [comment]
Closing as this branch seemed to build cleanly, pushed the code back to the orignal branch and merging #32 instead
